src/cosmology/cosmology_functions.cpp

```diff
@@ -20,29 +24,69 @@ void Grid3D::Initialize_Cosmology(struct Parameters *P)
   chprintf("Cosmology Successfully Initialized. \n\n");
 }
 
+Real Cosmology::dtda_cosmo(Real da, Real a)
+{
+  // Return dt/da * da
+  Real a2     = a * a;
+  Real fac_de = pow(a, -3 * (1 + w0 + wa)) * exp(-3 * wa * (1 - current_a));
+  Real a_dot  = sqrt(Omega_R / a2 + Omega_M / a + a2 * Omega_L * fac_de + Omega_K) * H0;
+  return da / a_dot;
+}
+
+Real Cosmology::Get_dt_from_da_rk(Real da, Real a)
+{
+  // Return dt/da * da
+  // But compute dt/da using a Runge-Kutta integration step
```

Comment:
Minor formatting suggesting in case we ever want to generate doxygen documentation

```suggestion

/* Compute dt/da * da. dt/da is computed with a Runge-Kutta integration step */
Real Cosmology::Get_dt_from_da_rk(Real da, Real a)
{
  
```

Reply:
I'll default to this format in the future.